conda-recipe/bld.bat

```diff
@@ -3,6 +3,20 @@
 @REM used BUILD_PREFIX as compiler installed in build section of meta.yml
 set "PATH=%BUILD_PREFIX%\Library\bin-llvm;%PATH%"
 
+REM A workaround for activate-dpcpp.bat issue to be addressed in 2021.4
```

Comment:
We are almost in 2024.1 is the workaround still applicable?

Reply:
It is there for dpctl, dpnp and dpbench. Even if it is not the case anymore, I would prefer to be consistent

Reply:
And since CI debugging takes lot of time, I would prefer not to experiment with it.